shared/js/background/trackers.es6.js

```diff
@@ -36,61 +36,59 @@ function isTracker (urlToCheck, thisTab, request) {
         }
     }
 
-    if (settings.getSetting('trackerBlockingEnabled')) {
```

Comment:
All I did here was remove the conditional above, so that we still count trackers and update the popup with them when tracker blocking is disabled.